src/test/java/org/opensearch/security/dlic/rest/api/TenantInfoActionTest.java

```diff
@@ -72,17 +79,78 @@ public void testTenantInfoAPIUpdate() throws Exception {
 
         // update security config
         RestHelper.HttpResponse response = rh.executePatchRequest(
-            BASE_ENDPOINT + "/api/securityconfig",
-            "[{\"op\": \"add\",\"path\": \"/config/dynamic/kibana/opendistro_role\"," + "\"value\": \"opendistro_security_internal\"}]",
-            new Header[0]
-        );
+                BASE_ENDPOINT + "/api/securityconfig",
+                "[{\"op\": \"add\",\"path\": \"/config/dynamic/kibana/opendistro_role\","
+                        + "\"value\": \"opendistro_security_internal\"}]",
+                new Header[0]);
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
 
-        response = rh.executePutRequest(BASE_ENDPOINT + "/api/rolesmapping/opendistro_security_internal", payload, new Header[0]);
+        response = rh.executePutRequest(BASE_ENDPOINT + "/api/rolesmapping/opendistro_security_internal", payload,
+                new Header[0]);
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
 
         rh.sendAdminCertificate = false;
         response = rh.executeGetRequest(ENDPOINT);
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
     }
+
+    @Test
+    public void testParallelPutRequests() throws Exception {
```

Comment:
Please rewrite this test case. See slack for context.

Reply:
Is this about the generative AI tools? This is confusing because I definitely did not use any AI tools to write this code. I did use code from UserApiTest.java for  the async request test helper functions, but I think that's fine to do so (please let me know if it's not). I just want to make it clear that I did not use any generative AI tools and will continue to not do so going forward. Thanks!

Reply:
Hi @prabhask5, thanks for the clarification. I will remove the block on the PR.